website/settings/defaults.py

```diff
@@ -227,7 +227,9 @@ def parent_dir(path):
 ALL_MY_REGISTRATIONS_ID = '-amr'
 ALL_MY_PROJECTS_NAME = 'All my projects'
 ALL_MY_REGISTRATIONS_NAME = 'All my registrations'
-NOTEWORTHY_LINKS_NODE = None
+POPULAR_LINKS_NODE = '8w6qe'  # Temporary.  Needs to be set to public node in production.
+NEW_AND_NOTEWORTHY_LINKS_NODE = 'su3q5' # Temporary. Needs to be set to public node in production.
+QA_USER_IDS = ['nxygz', 'x952z', 'tgak8', 'gaexu', 'rgc49', 'nsx26', 'j52af', 'rbk3c', 'xyubm', 'bje5z', 'twkqb', 'mvzr6', 'dihba'] # Should be updated periodically
```

Comment:
Picking up from https://github.com/pattisdr/osf.io/pull/2#discussion_r52221873 ...

This is better, but do we really have no way to configure our production environment other than this `defaults.py` file? I would expect to see `QA_USER_IDS = []` here, and then have some facility for setting this value in production (maybe `local.py`, maybe environment variables).